client.d.ts

```diff
@@ -2,4 +2,4 @@
 // be able to reference vite/client in project root.
 /// <reference types="vite/client" />
 
-export * from './dist/client'
+export * from './dist/client-types/index.js'
```

Comment:
dist/client-types is deleted after bundling

Reply:
@brc-dd Done!